numerus.go

```diff
@@ -132,10 +132,10 @@ func parse(s string) (uint, error) {
 
 // Parse takes a string in standard Roman Numeral notation and returns a Numeral.
 // If the given representation is invalid an error is returned.
-func Parse(s string) (Numeral, error) {
+func Parse(s string) (uint, error) {
```

Comment:
Not sure why this is changing?  Can't change the public API and break existing clients

Reply:
i change this line because when i try to convert roman num to number, it doesn't work

Reply:
Ok, so this is because you are calling `fmt.Println("parse roman num", num)` which will call `num.String()` under the hood, which will spit out the roman numeral format.  You need to case in the print statement for this to work, and not update the public API.

I've added examples to the tests to demonstrate this (see https://github.com/dhowden/numerus/blob/master/numerus_test.go#L135) along with some other changes I had locally, so I'm going to close this PR.